### PR TITLE
fix: MD050 consistent asterisk strong style

### PR DESCRIPTION
**Pull request by nschonni**
This is another one of those markdown vs. macros edge cases.
Will see if the escaping breaks the macro or not

### CODE REVIEW

**Comment by Josh-Cena** [comment]
Can we convert these to Markdown links instead?

**Comment by nschonni** [comment]
Looks like it doesn't like that escaping. i'll use the directives instead

**Comment by nschonni** [comment]
> Can we convert these to Markdown links instead?

That might be a better idea than the disable directives

**Comment by Josh-Cena** [comment]
Yes, since otherwise we would have the same struggle when formatting with Prettier.

File: files/en-us/mozilla/firefox/releases/39/index.md [hunk 1]
```diff
@@ -42,7 +42,7 @@ Highlights:
 ### JavaScript
 
 - The [`RegExp` constructor](/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp) no longer throws when the first argument is a `RegExp` and the second `flags` argument is present ({{bug(1108949)}}).
-- The {{jsxref("Object.noSuchMethod", "Object.prototype.__noSuchMethod__")}} property is now deprecated and throws a console warning (see {{bug(1140428)}} and this [platform announcement](https://groups.google.com/forum/#!topic/mozilla.dev.platform/0EkHgphxUo8)).
+- The `Object.prototype.noSuchMethod` property is now deprecated and throws a console warning (see {{bug(1140428)}} and this [platform announcement](https://groups.google.com/forum/#!topic/mozilla.dev.platform/0EkHgphxUo8)).
```
Comment (teoli2003) [inline files/en-us/mozilla/firefox/releases/39/index.md:45]:
```suggestion
- The `Object.prototype.__noSuchMethod__` property is now deprecated and throws a console warning (see {{bug(1140428)}} and this [platform announcement](https://groups.google.com/forum/#!topic/mozilla.dev.platform/0EkHgphxUo8)).
```

**Review by teoli2003 — APPROVED** [review]
👍

**Review by teoli2003 — APPROVED** [review]
Let's see how this goes. If it proves cumbersome for editors, we may have to revert, but let's try.

**Pull request merged**